src/Microsoft.Framework.PackageManager/Utils/FileOperationUtils.cs

```diff
@@ -38,12 +37,6 @@ private static void DeleteRecursive(string deletePath)
 
         public static bool MarkExecutable(string file)
         {
-            if (PlatformHelper.IsWindows)
```

Comment:
I removed this a while ago during an earlier iteration because I noticed everytime we call this function we already did the check for Windows. Would it be better to keep the check in here and remove it from the places we call it?